### ISSUE TITLE
Pin ViewComponent back to minor-level and troubleshoot the update

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/happy-heron#2198]
When ViewComponent released version 2.51.0, it broke our build. Specifically, it broke the `spec/components/collections/show/details_header_component_spec.rb` test, with the following error message:

```
Failure/Error: expect(rendered.css('turbo-frame')[1]['src']).to eq "/collection_versions/#{collection_version.id}/edit_link"
  Expected \e[33m"/collections/1068/edit_link"\e[0m to eq \e[35m"/collection_versions/1069/edit_link"\e[0m.
./spec/components/collections/show/details_header_component_spec.rb:18:in `block (3 levels) in <top (required)>'
```

At that time, we decided to pin ViewComponent to 2.50.0 (the prior version, which worked fine). Pin ViewComponent *back* to the minor-level and troubleshoot the error above. (Could be a bug in ViewComponent; could be a bug in how H2 uses ViewComponent.)

**Comment by jcoyne** [comment]
This seems to stem from https://github.com/github/view_component/issues/1315

**Comment by mjgiarlo** [comment]
Closed by #2204
